Makefile

```diff
@@ -1,137 +1,205 @@
-PHP_CS_FIXER_FUTURE_MODE=1
-PHPSTAN=./phpstan.phar
-PHP-CS-FIXER=./php-cs-fixer-v2.phar
-PHPUNIT=vendor/bin/phpunit
-INFECTION=build/bin/infection.phar
+.DEFAULT_GOAL := help
+
+.PHONY: help
+# TODO: add a check to ensure the result is always the expected one
```

Comment:
For this kind of TODO I think it would be best to add a PHPUnit test. I did that for example with PHP-Scoper and it works wonders, see [MakefileE2ETest.php](https://github.com/humbug/php-scoper/blob/188ff49/tests/MakefileE2ETest.php)